docs/Ignoring_files_from_build.md

```diff
@@ -0,0 +1,49 @@
+# Ignoring files from mbed build
+
+This is a documentation for `.mbedignore` file. It allows you to ignore files and directories from being processed by `mbed build` command.
```

Comment:
Skip the first sentence. Instead start: The `.mbedignore` file allows you to ignore...